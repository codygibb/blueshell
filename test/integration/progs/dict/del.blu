d := {'a': 5, 'b': 6, 'c': 7}
d.del('a')
print d
d.del('a') # Shouldn't produce error if key isn't there
