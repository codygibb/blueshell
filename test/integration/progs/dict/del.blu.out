{"b": 6, "c": 7}
